src/lib/atoms/strava.ts

```
@@ -0,0 +1,165 @@
+}>({
+  enabled: false,
+  attempts: 0,
+  maxAttempts: 3,
+  lastAttempt: null,
+})
+
+// Sync progress
+export const syncProgressAtom = atom<Record<string, unknown>>({})
+
+// Strava connection atom
+export const stravaConnectionAtom = atom<Record<string, any> | null>(null)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)